ksql-engine/src/test/java/io/confluent/ksql/planner/plan/KsqlStructuredDataOutputNodeTest.java

```diff
@@ -92,7 +93,7 @@ public void before() {
     props.put(KsqlConfig.SINK_NUMBER_OF_PARTITIONS_PROPERTY, 4);
     props.put(KsqlConfig.SINK_NUMBER_OF_REPLICAS_PROPERTY, (short)3);
     createOutputNode(props);
-    topicClient.createTopic(eq("output"), anyInt(), anyShort());
+    topicClient.createTopic(eq("output"), anyInt(), anyShort(), anyBoolean());
```

Comment:
We should add a unit test here that verifies that non windowed KTables actually are being created with `cleanup.polic=compact`.

Reply:
Added a unit test.